docs/platform/release-notes/tizen-5-5-m2.md

```diff
@@ -0,0 +1,641 @@
+  - MediaCodec
+    - The feature and exception tag of EventArgs class has been removed.
+  - MediaContent
+    - API to add face information has been added.
+  - MediaController
+    - API set to create instance without bundle have been added.
+    - API set to support subtitle, 360 mode, display mode, display rotation have been added.
+  - MediaTool
+    - API set to support PCM audio channel mapping have been added.
+  - MediaPlayer
+    - API set to export decoded audio, video data have been added.
+    - The cancelable version of PrepareAsync API has been added.
+    - API to get supported audio and video types has been changed. The API can get all supported codec types defined by each target.
+    - API to use audio offload has been added.
+  - MediaVision
+  - API set to support inference face detection, facial landmark detection, object detection, image classification have been added.
+- Multimedia Daemon
+  - Resource-manager
+    - New resource for audio offload has been added.
+
+### Fixes
+
+- C# API
+  - AudioIO
+    - Crash issue caused by using GC collected delegate has been fixed.
+    - Block and crash issue caused by thread deadlock have been fixed.
+  - AudioManager
+    - Crash issue caused by marshaling safe handle has been fixed.
+  - Camera
+    - Missed exception of constructor has been fixed.
+  - ImageUtil
+    - Crash issue caused by marshaling buffer has been fixed.
+    - Deprecated native p/invoke function has been changed.
+  - MediaPlayer
+- Crash issue caused by accessing null handle when enabling audio decided data has been fixed.
+- Multimedia Daemon
+  - Muse-server
+    - Security vulnerability has been fixed.
+    - Thread issues (scheduling and ASAN) when connecting instance has been fixed.
+    - Busy looping related with performance has been fixed.
+  - Resource-manager
+    - Forking failure of PIDfile has been fixed.
+    - Memory leak in heap consistency check has been fixed.
+    - Daemonizing issue related with GDBus has been fixed.
+- Wav Player
+  - WavPlayer
+    - Unexpected complete callback invocation in case of stopping playback has been fixed.
+- CVE patches
+  - libsndfile
+    - CVE-2019-3832
+    - CVE-2018-19758
+    - CVE-2018-19661 
+    - CVE-2018-19662
+    - CVE-2017-17456 
+    - CVE-2017-17457
+    - CVE-2017-14245
+    - CVE-2017-14246
+    - CVE-2017-8361
+    - CVE-2017-8363
+    - CVE-2017-8365
+
+### Known Issues
+
+- Media Vision
+  - Model data for inference API set are not provided. User can use model data in open model zoo.
+
+
+## Network and Connectivity
+
+### New and changed features
+
+- User awareness framework
+  - Combined condition of user detection for each service category has been introduced.
+    - Applications can define their interests of sensors as combined conditions of supported sensors.
+- Device provisioning protocol
+  - Wi-Fi Alliance’s Wi-Fi easy connect (DPP) specification has been developed.
```

Comment:
Wi-Fi easy connect for Wi-Fi Alliance (DPP) specification has been developed.

Reply:
I want to write "Wi-Fi easy connect (DPP) specification" as an combined expression.